src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/entity/TaskHistoryResource.java

```diff
@@ -10,6 +11,21 @@
 @Table(name = "task_history")
 public class TaskHistoryResource extends BaseTaskHistoryResource {
```

Comment:
This is your code but any ideas why there's a Base class being used here?  Seems unnecessary.

Reply:
That is to avoid code dupication in ReportableTaskResource, TaskHistoryResource